src/keas/kmi/testclient.py

```diff
@@ -50,31 +57,31 @@ def read_data(filename=None):
         return sys.stdin.read()
     else:
         try:
-            return file(filename, 'rb').read()
-        except IOError, e:
-            print >> sys.stderr, "Could not read %s" % filename
-            print >> sys.stderr, e
+            return open(filename, 'rb').read()
```

Comment:
A nice opportunity to use `with open(...) as fp: return fp.read()` -- it'll avoid ResourceWarnings on Python 3.